cli/Valet/AppArmor.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Valet;
+
+use Valet\Contracts\ServiceManager;
+
+class AppArmor
+{
+    public $sm;
+    public $cli;
+    public $files;
+    public $usrSbinDnsmasq;
+    public $phpFpm;
+    public $usrSbinAvahi;
+
+    /**
+     * Create a new Nginx instance.
+     *
+     * @param ServiceManager $sm
+     * @param CommandLine    $cli
+     * @param Filesystem     $files
+     * @return void
+     */
+    public function __construct(ServiceManager $sm, CommandLine $cli, Filesystem $files)
+    {
+        $this->cli = $cli;
+        $this->sm = $sm;
+        $this->files = $files;
+        $this->usrSbinDnsmasq = '/etc/apparmor.d/local/usr.sbin.dnsmasq';
+        $this->phpFpm = '/etc/apparmor.d/local/php-fpm';
+        $this->usrSbinAvahi = '/etc/apparmor.d/local/usr.sbin.avahi-daemon';
+    }
+
+    /**
+     * Install the configuration files for AppArmor.
+     *
+     * @return void
+     */
+    public function install()
+    {
+        if ((strpos($this->cli->run("aa-status"), 'is loaded') == 0)) {
```

Comment:
> Is this specific to debian/ubutun or is this behavior also on other distros (arch, ...)
OpenSUSE,Mandriva, too. In arch have the option to be active.
AppArmor is a module of kernel, so can be easy loaded in any distro, but cannot run with selinux active.

I added a condition to review if module is loaded, and now you mentioned I should implement in the construnct